tests/whitebox/op/wb_regex.cc

```diff
@@ -62,6 +62,6 @@ TEST_CASE( "pcre obj test", "[regex]" ) {
                 }
                 uint64_t l_dt_s;
                 l_dt_s = ns_waflz::get_delta_time_ms(l_t_s);
-                REQUIRE(l_dt_s < 200);
```

Comment:
So pcre2 is slow?

Reply:
I wrote a benchmark where I use the regexes from the OWASP rules and and it is 2-3x times faster than pcre1 but for this test it is slower

Reply:
do you have the benchmark tests handy? That sounds promising.

Reply:
@atn1990 @MichaelClayMoore and I are working on this. We'll update you soon